src/views/ManageList.jsx

```diff
@@ -1,7 +1,71 @@
-export function ManageList() {
+import { useState } from 'react';
+import { addItem } from '../api';
+
+export function ManageList({ listPath }) {
+	const [item, setItem] = useState({ name: '', urgency: 'soon' });
+	const [submitted, setSubmitted] = useState();
+
+	const handleChange = (e) => {
+		setItem({ ...item, [e.target.name]: e.target.value });
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		const { name, urgency } = item;
+
+		let nextPurchasedDate;
+		switch (urgency) {
+			case 'soon':
+				nextPurchasedDate = 7;
+				break;
+			case 'kindOfSoon':
+				nextPurchasedDate = 14;
+				break;
+			case 'notSoon':
+				nextPurchasedDate = 30;
+				break;
+			default:
+				console.log('Unrecognized selection');
+				return;
+		}
+		try {
+			await addItem(listPath, {
+				itemName: name,
+				daysUntilNextPurchase: nextPurchasedDate,
+			});
+			setSubmitted('added');
```

Comment:
suggestion - can we reset the form input fields after an item has been successfully added? Not necessary right now, but might be a nice user experience.